src/utils/getDefaultCommitContent.js

```diff
@@ -2,14 +2,23 @@
 import fs from 'fs'
 
 import { type CommitMode } from '../commands/commit'
+import configurationVault from './configurationVault'
+import { EMOJI_COMMIT_FORMATS } from '../commands/config/prompts'
 
 const COMMIT_FILE_PATH_INDEX = 3
 const COMMIT_TITLE_LINE_INDEX = 0
 const COMMIT_MESSAGE_LINE_INDEX = 2
 
-const getDefaultCommitContent = (
-  mode: CommitMode
-): { title: ?string, message: ?string } => {
+const gitmojiSymbolRegex = /^(\p{Emoji_Presentation})\s*(.*)/gu
+const gitmojiCodeRegex = /^(:[a-zA-Z_]+:)\s*(.*)/gu
```

Comment:
Why do we need this?

Reply:
To skip the gitmoji prompt I have to detect if the user has already filled it on the title part of the commit. This is the regex to do the job.